content/en/dashboards/functions/arithmetic.md

```diff
@@ -81,3 +87,5 @@ If a metric, `const_1{*}`, is a constant with the value of `1`, then `cumsum(con
     {{< nextlink href="/dashboards/functions/smoothing" >}}Smoothing: Smooth your metric variations.{{< /nextlink >}}
     {{< nextlink href="/dashboards/functions/timeshift" >}}Timeshift: Shift your metric data point along the timeline. {{< /nextlink >}}
 {{< /whatsnext >}}
+
+[1]: /monitors/types/configuration/?tab=thresholdalert#cumulative-time-windows
```

Comment:
Just noting that this link doesn't work for me.

Reply:
Ah, okay, in that case the link should be updated to instead point to `/monitors/configuration/?tab=thresholdalert#cumulative-time-windows`.

Reply:
modified!